src/UI/Wanted/Cutoff/CutoffUnmetLayout.js

```diff
@@ -229,11 +229,11 @@ module.exports = Marionette.Layout.extend({
     },
 
     _searchMissing  : function() {
-        if (window.confirm('Are you sure you want to search for {0} filtered missing movies?'.format(this.collection.state.totalRecords) +
+        if (window.confirm('Are you sure you want to search for {0} filtered missing movies? '.format(this.collection.state.totalRecords) +
                            'One API request to each indexer will be used for each movie. ' + 'This cannot be stopped once started.')) {
             CommandController.Execute('cutOffUnmetMoviesSearch', { name : 'cutOffUnmetMoviesSearch',
                                                            filterKey : this.collection.state.filterKey,
                                    filterValue : this.collection.state.filterValue });
         }
     },
-});
\ No newline at end of file
+});
```

Comment:
My editor made this change but I think it is sensible.

Reply:
It seems very sensible.